tests/dummy/app/styles/app.less

```diff
@@ -1 +1,61 @@
 @import 'oss-components';
+
+@keyframes slide-up {
+  0% {
+    opacity: 0;
+    transform: translate(-50%, 10%);
+  }
+
+  100% {
+    opacity: 1;
+    transform: translate(-50%, 0);
+  }
+}
+
+#toast-container.upf-toastr--container {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+  transform: translateX(-50%);
+
+  width: 100%;
+
+  .toast {
+    margin-bottom: @spacing-xx-sm;
+    max-width: 700px;
+    position: relative;
+    margin-left: 50%;
+    margin-right: 50%;
+    transform: translateX(-50%);
+    align-items: center;
+  }
+
+  .toast--hidden {
+    display: flex;
+    opacity: 0;
+    animation: none;
+  }
```

Comment:
Is it better to use `visibility: hidden`?

Reply:
FYI, this is probably linked to the animation state. You can't have a proper show/hide CSS animation on the "visibility" attribute whereas it works nicely over "opacity".

Reply:
all as he said :p